src/components/Grid/Grid.stories.mdx

```diff
@@ -1,27 +1,52 @@
 Codelitt’s grid system uses a series of containers, rows, and columns to layout and align content. It’s built with flexbox and
 is fully responsive. Below is an example and an in-depth look at how the grid comes together.
-Column classes indicate the number of columns you’d like to use out of the possible 12 per row or less depending on the screen size:
+Column classes indicate the number of columns you’d like to use out of the possible 16 per row or less depending on the screen size:
 
+* `xl` and `xxl`: Extra Large screens (desktops) are divided into 16 columns.
```

Comment:
Can you please confirm this? I believe that the `16` columns are only for the fluid containers, the centered ones keep 12 columns no matter what